Sources/PMKFoundation.h

```diff
@@ -1,6 +1,3 @@
 #import "NSNotificationCenter+AnyPromise.h"
 #import "NSURLSession+AnyPromise.h"
-
-#if TARGET_OS_MAC && !TARGET_OS_EMBEDDED && !TARGET_OS_SIMULATOR
-    #import "NSTask+AnyPromise.h"
-#endif
```

Comment:
This broke CocoaPods. Sigh.

Supporting three different ways to install PromiseKit sucks. Can't easily add tests for this either.